lib/chewy/index/adapter/mongoid.rb

```diff
@@ -0,0 +1,87 @@
+require 'chewy/index/adapter/orm'
+
```

Comment:
Mongoid support was dropped in Chewy 7. We have added the adapter again along with feature parity from our current forked version